pangea-sdk/v3/service/authz/api.go

```diff
@@ -37,68 +37,62 @@ type TupleCreateResult struct {
 //
 // @description Write tuples. The request will fail if tuples do not validate against the schema defined resource types.
 //
-// @operationId authz_post_v1beta_tuple_create
+// @operationId authz_post_v1_tuple_create
 //
 // @example
 //
 //	rCreate, err := cli.TupleCreate(ctx, &authz.TupleCreateRequest{
 //		Tuples: []authz.Tuple{
 //			authz.Tuple{
```

Comment:
Should this line be? It was removed from the example in `examples` folder

Reply:
It's optional.